language-support/scala/bindings/src/main/scala/com/digitalasset/ledger/client/binding/binding/TemplateCompanion.scala

```diff
@@ -81,6 +81,7 @@ abstract class TemplateCompanion[T](implicit isTemplate: T <~< Template[T])
   }
 
   protected final def ` exercise`[ExOn, Out](
+      actor: Primitive.Party,
```

Comment:
This is not part of the public API right? so this is not a breaking change

Reply:
Right.